examples/data-sources/bloxone_dns_records/data-source.tf

```diff
@@ -0,0 +1,16 @@
+# Get records filtered by an attribute
+data "bloxone_dns_records" "example_by_attribute" {
```

Comment:
would this handle the `generic` record in addition to everything?

Reply:
No, it is supposed to handle only types not supported directly. I have added a `type` field in the data source for this.